lib/repl.js

```diff
@@ -1256,13 +1256,16 @@ function defineDefaultCommands(repl) {
       try {
         var stats = fs.statSync(file);
         if (stats && stats.isFile()) {
+          this.editorMode = true;
+          REPLServer.super_.prototype.setPrompt.call(this, '');
```

Comment:
this is just so it'll be "pretty", right?

Reply:
It would feel natural to have the `enableEditorMode` but I think this should not block the PR from landing.